galaxy/templates/ingress-activity-canary.yaml

```diff
@@ -39,9 +39,17 @@ spec:
         paths:
           - path: {{ $ingressPath }}/api/users/
             pathType: Prefix
+{{- if $.Capabilities.APIVersions.Has "networking.k8s.io/v1" }}
+            backend:
+              service:
+                name: {{ $fullName }}-nginx
+                port:
+                  number: {{ $servicePort }}
+{{- else -}}
             backend:
```

Comment:
Shouldn't this change for `networking.k8s.io/v1beta1` vs `extensions/v1beta1` ?
I'm guessing just removed since should be same as `networking.k8s.io/v1`?